tests/kernel/install_kotd.pm

```diff
@@ -102,15 +103,26 @@ END
 
 sub run {
     my $self = shift;
+
+    my $baseurl;
+    my $default_url;
+    my $url;
+    my $version;
+
     $self->wait_boot;
 
     $self->select_serial_terminal;
 
-    my $url = get_required_var("INSTALL_KOTD");
-    if ($url !~ /^https?:\/\//) {
-        my $arch = get_required_var("ARCH");
-        $url = "http://download.suse.de/ibs/Devel:/Kernel:/$url/standard/$arch/";
+    if (is_sle) {
+        $version = "SLE" . get_required_var("VERSION");
+        $baseurl = "http://download.suse.de/ibs/Devel:";
+    } else {
+        $version = "HEAD";
+        $baseurl = "http://download.opensuse.org/repositories";
     }
+    $default_url = "$baseurl/Kernel:/$version/standard";
+    $url         = get_var("KOTD_REPO", $default_url);
+    $url .= "/" . get_required_var("ARCH") . "/";
```

Comment:
Will it really work for other ARCHs?  Repositories are:

- http://download.opensuse.org/repositories/Kernel:/HEAD/ARM/
- http://download.opensuse.org/repositories/Kernel:/HEAD/PPC/

Reply:
it's pointing to 'standard' dir http://download.opensuse.org/repositories/Kernel:/HEAD/standard/

Reply:
Yes, but code is giving impression that it works for multiple architectures, but it is true only for SLE. openSUSE repository has slightly different layout for ppc and arm.

Reply:
Agree, openSUSE doesn't work for other archs (there is i586 and i686, but we don't care). I don't have neither ppc and arm, so cannot test it. I suggest to ignore this and enable KOTD only for intel on openSUSE just now.

Reply:
Fine, we can address it in the future.

Reply:
@czerw OK, addressed arm and ppc on openSUSE. Uh, but you were faster :)

Reply:
But thanks, I can enable jobs now for openSUSE